apps/dashboard/src/pages/workflows.tsx

```diff
@@ -202,7 +202,7 @@ export const WorkflowsPage = () => {
                       }}
                     >
                       <RiFileMarkedLine />
-                      View Workflow Gallery
```

Comment:
Copywriting fix: It's the Workflow Template store, we don't use the term Gallery.